src/gitcomp/ser_de.py

```diff
@@ -141,7 +144,7 @@ def __get_entries_as_rows(g: Dict[str, Any]) -> List[Any]:
     @staticmethod
     def __get_transpose(g: Dict[str, Union[User, Repository]], rows, headers):
         new_rows = []
-        new_headers = ['Argument'] + list(g.keys())
+        new_headers = ['Argument'] + sorted(list(g.keys()))
```

Comment:
We should avoid sorting the usernames as the order may convey some relevant information. Try changing the rows instead